js/utils.js

```diff
@@ -460,6 +611,19 @@ Fliplet.Registry.set('dynamicListUtils', (function() {
           return splitByCommas(filter.value).includes(rowData);
```

Comment:
`Array.prototype.includes()` [doesn't worked in IE](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes).